src/Board/List/TasksCards/Description/Description.js

```diff
@@ -0,0 +1,133 @@
+import React, { useState } from "react";
+import { useParams, Link } from "react-router-dom";
+import { Card } from "antd";
+import { useSelector, useDispatch } from "react-redux";
+import { Form, Input, Button } from "antd";
+import {
+  createDescription,
+  deleteDescription,
+} from "../../../../Data/TaskReducer";
+import s from "./Description.module.css";
+import PageNotFound from "../../../PageNotFound";
+
+export let ContainerDescription = () => {
+  let id = useParams().descriptionId;
+  let tasks = useSelector((state) => state.tasks).find((el) =>
+    el.find((item) => item.id === id)
+  );
+  let content = !tasks ? <PageNotFound /> : <Description tasks={tasks} />;
+  return content;
+};
+
+let Description = ({ tasks }) => {
+  const [form] = Form.useForm();
+  const dispatch = useDispatch();
+
+  let id = useParams().descriptionId;
```

Comment:
Description is not separate entity. It is a part of task. So there should not be any ids for descriptions